h/static/scripts/drafts.js

```diff
@@ -31,47 +39,53 @@ function DraftStore() {
    * unsaved drafts exist.
    */
   this.unsaved = function unsaved() {
-    return this._drafts.filter(function (draft) {
+    return this._drafts.filter(function(draft) {
       return !draft.model.id;
-    }).map(function (draft) {
+    }).map(function(draft) {
       return draft.model;
     });
-  }
+  };
 
   /** Retrieve the draft changes for an annotation. */
   this.get = function get(model) {
-    for (var i=0; i < this._drafts.length; i++) {
+    for (var i = 0; i < this._drafts.length; i++) {
       if (match(this._drafts[i], model)) {
-        return this._drafts[i].changes;
+        return {
+          private: this._drafts[i].private,
+          tags: this._drafts[i].tags,
+          text: this._drafts[i].text
+        };
       }
     }
-  }
+  };
 
   /**
    * Update the draft version for a given annotation, replacing any
    * existing draft.
    */
-  this.update = function update(model, changes) {
+  this.update = function update(model, private, tags, text) {
     var newDraft = {
       model: model,
-      changes: changes,
+      private: private,
+      tags: tags,
+      text: text
     };
     this.remove(model);
     this._drafts.push(newDraft);
-  }
+  };
 
   /** Remove the draft version of an annotation. */
   this.remove = function remove(model) {
-    this._drafts = this._drafts.filter(function (draft) {
+    this._drafts = this._drafts.filter(function(draft) {
       return !match(draft, model);
     });
-  }
+  };
 
   this.discard = function discard() {
     this._drafts = [];
-  }
+  };
 }
 
-module.exports = function () {
+module.exports = function() {
```

Comment:
Should have a space after the `function` keyword for consistency with the rest of our code.